src/Analyzers/CSharp/Tests/UseCompoundAssignment/UseCompoundCoalesceAssignmentTests.cs

```diff
@@ -64,7 +73,7 @@ public async Task TestRightMustBeParenthesized()
 @"class Program
 {
     private static string s_goo;
-    private static string Goo => s_goo [||]?? s_goo = new string('c', 42);
+    private static string Goo => {|CS0131:s_goo ?? s_goo|} = new string('c', 42);
```

Comment:
because the original code is invalid C#.  the test is testing that.  the new test harness requires you to be explicit htat you know it's invalid code by putting in these markers.  it helps prevent tests that are accidentally testing bogus code unintentionally.